lib/licensee/license.rb

```diff
@@ -63,7 +63,7 @@ def licenses
     # `no-license` - The project is not licensed (e.g., all rights reserved)
     #
     # Note: A lack of detected license will be a nil license
-    PSEUDO_LICENSES = %w[other no-license].freeze
+    PSEUDO_LICENSES = %w(other no-license).freeze
```

Comment:
Thanks, I'm pretty unfamiliar with Ruby, I deleted my ```Gemfile.lock``` and reinstalled the dependencies and yes that made those lint errors go away, I've reverted those changes.